ReleaseProcessAutomation/ReleaseProcessAutomation.csproj

```
@@ -1,17 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <PackageId>ReleaseProcessAutomation</PackageId>
-        <Version>1.0.0</Version>
-        <Authors>re-motion</Authors>
-        <Company>RUBICON</Company>
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
         <PackAsTool>true</PackAsTool>
         <ToolCommandName>rpa</ToolCommandName>
         <Nullable>enable</Nullable>
         <RootNamespace>ReleaseProcessAutomation</RootNamespace>
     </PropertyGroup>
+    
+    <PropertyGroup>
+        <PackageId>Remotion.ReleaseProcessAutomation</PackageId>
+        <Version>1.0.0</Version>
+        <Authors>rubicon IT GmbH</Authors>
+        <Owners>rubicon IT GmbH</Owners>
+        <developmentDependency>true</developmentDependency>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <PackageProjectUrl>http://www.re-motion.org/</PackageProjectUrl>
+        <Description>The re-motion Release Process Automation tool provides automation of the release process with Jira, Git and MSBuild</Description>
+        <Copyright>Copyright (c) rubicon IT GmbH, www.rubicon.eu - Licensed under Apache License, Version 2.0</Copyright>
+        <PackageTags>re-motion MSBuild</PackageTags>
+        <PackageTags>re-motion Release
+        </PackageTags>
```

The `Release` and `Debug` tags should indicate the difference between the configuration used to build the package. Normally this would be done by the build script but since we can't use it yet we have to do it ourselves:

```suggestion
        <PackageTags>re-motion</PackageTags>
        <PackageTags Condition="'$(Configuration)' == 'Debug'">$(PackageTags) Debug</PackageTags>
        <PackageTags Condition="'$(Configuration)' == 'Release'">$(PackageTags) Release</PackageTags>
```

---

`Release` was not supposed to show whether it was built as debug or release, just that it helps with releasing. Changed to ReleaseAutomation so that it is more understandable

---

Hmm... ReleaseAutomation is already sort of part of the package name. how about just simplify it to "re-motion Tooling"? The old script had "re-motion Powershell Git" but tbh, "Powershell" had some meaning, "Git" looks a bit iff from 10 years distance.